src/Illuminate/Contracts/Support/RegistrableProvider.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Illuminate\Contracts\Support;
+
+interface RegistrableProvider
```

Comment:
Shouldn't it be `RegisterableProvider`?

Reply:
I think `RegistrableProvider` is correct.